src/vm/prestub.cpp

```diff
@@ -2947,13 +2947,15 @@ PCODE DynamicHelperFixup(TransitionBlock * pTransitionBlock, TADDR * pCell, DWOR
         {
         case ENCODE_NEW_HELPER:
             {
-                CorInfoHelpFunc helpFunc = CEEInfo::getNewHelperStatic(th.AsMethodTable());
+                bool hasSideEffects;
+                CorInfoHelpFunc helpFunc = CEEInfo::getNewHelperStatic(th.AsMethodTable(), &hasSideEffects);
```

Comment:
Nit: since there are multiple places like this (the caller doesn't care about the new flag) - it might be worth considering allowing null as the new parameter (which means "don't care"). The pattern is pretty common in JitInterface.

Reply:
Done.